packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -976,7 +988,8 @@ export class InputManager {
     }
 
     /**
-     * @param mesh
+     * @param mesh - Mesh to invalidate
+     // eslint-disable-next-line tsdoc/syntax

```

Comment:
Do we have to do this to all `@hidden` functions?

Reply:
Just asked in chat and we do not.  Removed